config/zfs-build.m4

```diff
@@ -274,20 +274,20 @@ AC_DEFUN([ZFS_AC_RPM], [
 	dnl #	./configure --with-config=srpm
 	dnl #
 	AS_IF([test -n "$dracutdir" ], [
-		RPM_DEFINE_UTIL='--define "_dracutdir $(dracutdir)"'
+		RPM_DEFINE_UTIL=${RPM_DEFINE_UTIL}' --define "_dracutdir $(dracutdir)"'
```

Comment:
@hvdijk Was going from `=` to (basically) `+=` on purpose here?

Reply:
Yes. In the past, `_dracutdir`, `_udevdir`, `_udevruledir` and `_initconfdir` would all be defined. Commit 9ea6c3d made it so that `_initconfdir` would no longer be defined if `_dracutdir` got defined, but per that commit's commit message, that was an accident, the intent had been to use `+=` here.